### PR TITLE
Add Acquia Lando support

### PR DESCRIPTION
**Pull request by grasmash**

### CODE REVIEW

**Comment by mikemadison13** [comment]
the previous iteration of this patch worked. the current version does not w/o me further modifying so that:

```
 /**
   * If this isn't a Cloud environment, assume it's local.
   */
  public static function isLocalEnv() {
    return !self::isAhEnv() || self::isAcquiaLandoEnv() || self::isLandoEnv();
  }
```

it seems that isLandoEnv works as expected but isAcquiaLandoEnv does not.

**Comment by grasmash** [comment]
@mikemadison13 What's the value of `AH_SITE_ENVIRONMENT` in your Lando env?

**Comment by mikemadison13** [comment]
@grasmash it's reporting as LANDO.

**Comment by grasmash** [comment]
Fixed, maybe.

File: src/AcquiaDrupalEnvironmentDetector.php [hunk 1]
```diff
@@ -222,11 +222,32 @@ public static function getSiteName($site_path) {
...
+  public static function getLandoInfo() {
+    return getenv('LANDO_INFO');
+  }
+
   /**
    * If this isn't a Cloud environment, assume it's local.
    */
   public static function isLocalEnv() {
-    return !self::isAhEnv();
+    return !self::isAhEnv() || self::isAcquiaLandoEnv();
```
Comment (danepowell) [inline src/AcquiaDrupalEnvironmentDetector.php:250]: I'm fine with this change, but is it intentional? Last I heard it was a problem if `isLocalEnv()`returns true in Lando.

If this works, then awesome.

Comment (grasmash) [reply src/AcquiaDrupalEnvironmentDetector.php:250]: Yes this is intentional. @mikemadison13 confirmed this works.

**Comment by dhaggman** [comment]
Can you do a tag release with this?

**Comment by danepowell** [comment]
Sure: https://github.com/acquia/drupal-environment-detector/releases/tag/v1.4.0

**Pull request merged**